packages/amazon-sumerian-hosts-babylon/examples/gestures-demo.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gestures Demo</title>
+
+    <!-- Latest compiled and minified CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@3.4.1/dist/css/bootstrap.min.css"
+      integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="gesture-demo.css" />
+
+    <!-- Babylon.js dependencies -->
+    <script src="https://cdn.jsdelivr.net/npm/babylonjs@4.2.1/babylon.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/babylonjs-loaders@4.2.1/babylonjs.loaders.min.js"></script>
+
+    <!-- Sumerian Host dependencies -->
+    <script
+      type="text/javascript"
+      src="https://sdk.amazonaws.com/js/aws-sdk-2.645.0.min.js"
+    ></script>
+    <script type="text/javascript" src="../dist/host.babylon.js"></script>
```

Comment:
i think we could remove this src since we're using ES6 module for the script tag below which should also enable the live loading feature of webpack dev server

Reply:
Although I happen to be using webpack-dev-server to make previewing the example site easier, I don't want a hard dependency on that server. I want to the examples to work if they're deployed to other servers as well (like Amplify hosting). Would your suggested change represent a hard dependency on webpack's server?

Reply:
Sorry i didn't look clearly about how you structure files, it looks like we're just breaking down example source scripts. I'm ok to leave as this right now.